src/lib/sitemap.ts

```diff
@@ -240,12 +243,13 @@ export function generatePaths(
   // See: https://kit.svelte.dev/docs/advanced-routing#advanced-layouts-breaking-out-of-layouts
   let routes = Object.keys(import.meta.glob('/src/routes/**/+page*.svelte'));
 
-  // Validation: if dev has one or more routes that start with `[[lang]]`,
+  // Validation: if dev has one or more routes that have the lang parameter
```

Comment:
`have the lang parameter` -> `contain a lang parameter, optional or required,`

for clarity